packages/components/src/form-token-field/test/index.tsx

```diff
@@ -950,6 +950,88 @@ describe( 'FormTokenField', () => {
 			expect( onChangeSpy ).not.toHaveBeenCalled();
 		} );
 
+		it( 'should hide the suggestion list on blur and empty input', async () => {
```

Comment:
Noting that this test would pass also without the changes proposed by this PR.

Actually, the suggestion list should not be visible already after the input is cleared. Blurring the input in this case shouldn't make any differences.

Could we tweak this unit test by removing the `await user.click( document.body );` action and updating the test name to something like "should hide the suggestion list on an empty input" ?